### PR TITLE
fix(curriculum): add description for C# exam

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 1]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
 
 # --description--
 
-Here are some rules:
+Pass this exam to earn your Foundational C# with Microsoft Certification. Please familiarize yourself with the following guidelines and click the start button when you are ready to begin:
 
-- click start
+- You must complete the exam in a single session.
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:12]:
```suggestion
- You must complete the entire exam in one session.
```

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 2]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
...
+- If you exit the exam at any point, it will result in the loss of your progress.
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:13]:
```suggestion
- Exiting the exam at any point will result in the loss of your progress.
```

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 3]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
...
+- The exam will comprise of multiple-choice questions.
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:14]:
```suggestion
- The exam consists of multiple-choice questions.
```

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 4]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
...
+- There is no time limit, but your time will be recorded.
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:15]:
```suggestion
- There is no specific time limit, but your total time taken will be recorded.
```

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 5]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
...
+- Complete the exam and answer a sufficent number of question correctly to qualify to claim your certification.
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:16]:
```suggestion
- To qualify for certification, you need to complete the exam and answer a sufficient number of questions correctly.
```

File: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md [hunk 6]
```diff
@@ -7,9 +7,13 @@ dashedName: foundational-c-sharp-with-microsoft-certification-exam
 
 # --description--
 
-Here are some rules:
+Pass this exam to earn your Foundational C# with Microsoft Certification. Please familiarize yourself with the following guidelines and click the start button when you are ready to begin:
```
Comment (Sembauke) [inline curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md:10]:
```suggestion
To earn your Foundational C# with Microsoft Certification, you need to pass this exam. Before starting the exam, please review the following guidelines:
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Comment by moT01** [comment]
Thanks for taking a look @Sembauke 👍 These suggestions didn't seem particularly necessary. Some were maybe a little better or maybe a little worse in cases. The wording seems more like a preference - I think they worked either way. I ended up adding many of them (with slight modifications). Hopefully, it's good enough 👍

**Comment by Sembauke** [comment]
No problem @moT01, that's were the reviews are for.

**Review by Sembauke — APPROVED** [review]

**Review by Sboonny — APPROVED** [review]
LGTM

**Pull request merged**